docs/4_preparing_roms_input_datasets.ipynb

```diff
@@ -1,5 +1,18 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a81a07fe",
+   "metadata": {},
+   "source": [
+    "> [!Warning] \n",
+    "> **This project is still in an early phase of development.**\n",
+    ">\n",
+    "> The [python API](https://c-star.readthedocs.io/en/latest/api.html) is not yet stable, and the schema for the [blueprint](https://c-star.readthedocs.io/en/latest/terminology.html#term-blueprint) will likely evolve. \n",
+    "> Therefore whilst you are welcome to try out using the package, we cannot yet guarantee backwards compatibility. \n",
+    "> We expect to reach a more stable version in Q1 2025."
+   ]
+  },
   {
```

Comment:
There is no use of C-Star in this notebook at all. It is basically a quickstart for ROMS-Tools. Should we adapt the language of the warning?

Reply:
I think the point was just to stick the warning on every single page of the docs and examples so that it can't possibly be missed.

Reply:
Fair enough.

Reply:
I mean this notebook is still about preparing datasets for use in C-Star - I would still want to know about the warning before trying to run this.

Reply:
I guess this notebook also assumes they work on Perlmutter due to this line and the use of "PSCRATCH":
```
target_dir = Path(os.getenv("PSCRATCH")) / "ROMS_TOOLS_INPUT_DATA"
```
We should change this machine-dependency (later) because otherwise the notebook is machine-dependent. For now, we could add the "Perlmutter warning".